web/src/tippyjs.js

```diff
@@ -429,6 +429,18 @@ export function initialize() {
         },
     });
 
+    delegate("body", {
+        target: [".generate_multiuse_invite_radio_label.disabled_setting_tooltip"],
+        content: $t({
+            defaultMessage:
+                "You do not have permissions to generate invite links in this organization.",
```

Comment:
The tooltip is only shown when hovering over the label. I think we should show it when hovering over the radio button as well, basically in the whole row from radio button to end of the label text.